packages/react-components/src/components/UserGuide/UserGuide.tsx

```diff
@@ -0,0 +1,247 @@
+import { FC, PropsWithChildren, useEffect, useRef, useState } from 'react';
+
+import { FloatingPortal } from '@floating-ui/react';
+import { autoUpdate, flip, shift, useFloating } from '@floating-ui/react-dom';
+import cx from 'clsx';
+
+import { IUserGuide } from './types';
+import VirtualReference from './virtualElementReference';
+
+import styles from './UserGuide.module.scss';
+
+const baseClass = 'user-guide';
+
+const virtualReference = (element: Element, padding: number = 0) =>
+  new VirtualReference(element, padding);
+
+export const UserGuide: FC<PropsWithChildren<IUserGuide>> = ({
+  className,
+  children,
+  cursorPosition = 'bottom',
```

Comment:
In this case, do we completely abandon the previous look and behavior of UserGuide?

I ask about this because Agent App uses the current UserGuide component in its current app tours (which will stay), so this is partly a breaking change and a possible regression for all tours in the application

Reply:
AFAIK all tours should be verified so this breaking change should go hand in hand with this verification. 🤷

Reply:
Wouldn't it be better to prepare new component like `v2` in this case? I can see that there are many tours to check:
`export const enum UserGuidedTourType {
  ChatCannedResponses = 'ChatCannedResponses',
  CustomersList = 'CustomersList',
  CustomSegments = 'CustomSegments',
  TicketPrivacy = 'TicketPrivacy',
  TicketFormSettings = 'TicketFormSettings',
  TicketStatus = 'TicketStatus',
  TagsUsage = 'TagsUsage',
  TargetedMessages = 'TargetedMessages',
  HomeScreenExplore = 'HomeScreenExplore',
  OpenChat = 'OpenChat',
  UnassignedChat = 'UnassignedChat',
  RichGreeting = 'RichGreeting',
  ReportViews = 'ReportViews',
  RoutingRules = 'RoutingRules',
  Traffic = 'Traffic',
}`

Reply:
☝️ What's the reasoning for changing the current component and not creating the V2 version like with PromoBanner? It automatically creates dependencies and possible blockers for product teams to introduce this

Reply:
@MrVorgoth @MateuszRomek  I think most of these tours might have only one step. From my experience, we tend to forget about tours and don't test them - so in my opinion, it's a good idea to introduce one breaking change and clean up the whole app. Recently, I deleted two or three tours just because they didn't make sense. Let's check the list; for example, I see that tickets should be removed as we will replace tickets with helpdesk.

Reply:
@dprzybylek it's not about changing these tours but it's design decision. Marta will talk to Vlad about this as this must be product designers decision to make all of the tours like this onboarding tour and remove tooltip approach:) 
AFAIK there were no discussions about this